.github/workflows/benchmarkdotnet.yml

```diff
@@ -32,3 +33,18 @@ jobs:
           comment-on-alert: true
           fail-on-alert: true
           alert-comment-cc-users: '@ktrz'
+
+      - name: Store benchmark result - separate results repo
+        uses: rhysd/github-action-benchmark@v1
+        with:
+          name: Benchmark.Net Benchmark
+          tool: 'benchmarkdotnet'
+          output-file-path: examples/benchmarkdotnet/BenchmarkDotNet.Artifacts/results/Sample.Benchmarks-report-full-compressed.json
+          github-token: ${{ secrets.BENCHMARK_ACTION_BOT_TOKEN }}
+          auto-push: true
+          # Show alert with commit comment on detecting possible performance regression
+          alert-threshold: '200%'
```

Comment:
is this number ok ? not sure about it XD

Reply:
This is just an example so it doesn't matter that much. The value is the same as above. I think that this step won't work anyway cause of the token not being present and repo being just a clone. I'll comment it out